server/src/main/java/org/elasticsearch/transport/RemoteClusterService.java

```diff
@@ -119,6 +119,12 @@ public final class RemoteClusterService extends RemoteClusterAware implements Cl
         )
     );
 
+    public static final Setting.AffixSetting<String> REMOTE_CLUSTER_AUTHORIZATION = Setting.affixKeySetting(
+        "cluster.remote.",
+        "authorization",
+        key -> Setting.simpleString(key, v -> {}, Setting.Property.Dynamic, Setting.Property.NodeScope, Setting.Property.Filtered)
```

Comment:
do we want any validation on this setting ? for example, ensuring that it can be Base64 decoded to a string that has a colon.

Reply:
Encoding is deferred to future PR. This is just adding the setting as an opaque string, I think there is still open discussion about whether what the FC will send to QC. Just an API Key, or an enrollment token that possibly combined the API Key with metadata, for example.